### DISCUSSION TITLE
How to add favicon in the swagger docs?

### DISCUSSION THREAD

**Discussion by kulterryan** [elysiajs/elysia#699] [General]
I just want to understand, how can we add a favicon in the swagger docs so that I does not return any 404 error?

<img width="578" alt="Screenshot 2024-07-02 at 4 15 33 PM" src="https://github.com/elysiajs/elysia/assets/36154235/57088433-3bda-4781-80ab-8ad21b45442f">

**Comment by mtt-artis** [comment]
Hey 👋 
You need to add a route to serve the favicon file

```ts
new Elysia()
    .get('/favicon.ico', Bun.file('./statics/favicon.ico'));
```

Or you can use the static plugging https://elysiajs.com/plugins/static.html
